src/basic/mount-util.h

```diff
@@ -29,6 +29,14 @@
 #include "macro.h"
 #include "missing.h"
 
+#define SD_INACCESSIBLE_PATH "/run/systemd/inaccessible/"
+#define SD_INACCESSIBLE_PATH_REG SD_INACCESSIBLE_PATH"reg"
+#define SD_INACCESSIBLE_PATH_DIR SD_INACCESSIBLE_PATH"dir"
+#define SD_INACCESSIBLE_PATH_BLK SD_INACCESSIBLE_PATH"blk"
+#define SD_INACCESSIBLE_PATH_CHR SD_INACCESSIBLE_PATH"chr"
+#define SD_INACCESSIBLE_PATH_FIFO SD_INACCESSIBLE_PATH"fifo"
+#define SD_INACCESSIBLE_PATH_SOCK SD_INACCESSIBLE_PATH"sock"
```

Comment:
I really don't see the point of making this a macro, in particular a global one. I mean, this just adds one extra indirection step, and harder to grok. Just using "/run/systemd/inaccessible/reg" directly at the two places we need this makes this much more readable I would say.

Moreover we use "SD_" as prefix only in exported APIs of libsystemd (where namespacing matters). As this is an internal header this shouldn't have that.